src/python/system/process_handler.py

```diff
@@ -500,14 +500,17 @@ def terminate_stale_application_instances():
 
   additional_process_to_kill = environment.get_value(
       'ADDITIONAL_PROCESSES_TO_KILL')
-  app_name = environment.get_value('APP_NAME')
   builds_directory = environment.get_value('BUILDS_DIR')
   llvm_symbolizer_filename = environment.get_executable_filename(
       'llvm-symbolizer')
   platform = environment.platform()
   start_time = time.time()
 
-  processes_to_kill = [app_name]
+  processes_to_kill = []
+  if environment.get_value('KILL_PROCESS_MATCHING_APP_NAME', True):
```

Comment:
This will likely need a better long-term fix (e.g. running the tool as another user or isolated in some other way), but if we assume users are running the tool directly it's fairly annoying to have it constantly terminate "chrome".